src/@types/parse-duration/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'parse-duration' {
```

Comment:
Why are you adding type declarations when it has typings? I don't think they are needed. I have already installed the package for my branch as well and everything works alright...

Reply:
Hmm..
WIll double check